matterhorn/Dockerfile

```diff
@@ -1,30 +1,32 @@
-FROM        phusion/baseimage:latest
+FROM        phusion/baseimage:master
```

Comment:
Is this a safe change or does it make things more unstable?

Reply:
master is phusion's most up-to-date and stable. latest was several generations behind.